glsl/combined.frag

```diff
@@ -205,6 +274,9 @@ void main()
 	if (c.a == 0.0)
 		discard;
 
+	if (vRGBAFraction.r > 0 && vTexMetadata.s > 0)
```

Comment:
`0.0` to avoid

> ERROR: 0:277: '>' : wrong operand types - no operation '>' exists that takes a left-hand operand of type 'mediump float' and a right operand of type 'const int' (or there is no acceptable conversion)

Reply:
Fixed.